### PR TITLE
Adds config and prompt for task runner

### PR DESCRIPTION
**Pull request by Toddses**
Removes the assumption that the user is using Grunt, and adds the option to use Grunt or Gulp as the task runner, with Grunt as the default.

### CODE REVIEW

File: app/prompts.js [hunk 1]
```diff
@@ -160,6 +160,19 @@ module.exports = function(advanced, defaults) {
 				return !!res.installTheme;
 			}
 		}, {
+			message: 'Task runner (grunt/gulp)',
+			name: 'themeTaskRunner',
+			default: defaults.themeTaskRunner || 'grunt',
+			validate: function(value) {
+				if (value != '' && /^(?:grunt|gulp)$/.test(value)) {
```
Comment (wesleytodd) [inline app/prompts.js]: We might want this to be case insensitive

Comment (Toddses) [inline app/prompts.js]: for sure.

**Comment by wesleytodd** [comment]
Once you add that I will get this in for the big 2.0 release.

**Comment by Toddses** [comment]
Before we pull this in, I'm going to add a filter so it always stores in lowercase. I'll do the same for the git/tar option when pulling in a custom theme as well.

**Comment by Toddses** [comment]
Should be good to go here.

File: app/prompts.js [hunk 2]
```diff
@@ -151,15 +151,34 @@ module.exports = function(advanced, defaults) {
...
+			name: 'themeTaskRunner',
+			default: defaults.themeTaskRunner || 'grunt',
+			validate: function(value) {
+				if (value != '' && /^(?:grunt|gulp)$/i.test(value)) {
+					return true;
+				}
+				return false;
+			},
+			filter: function(value) {
+				return value.toLowerCase();
```
Comment (wesleytodd) [inline app/prompts.js]: Adding the filter here pretty much makes the case insensitive regex unnecessary.  But I can fix this once I merge it in.

**Comment by wesleytodd** [comment]
Merged this one despite the comment, but if you feel like changing that then be my guest.

**Comment by Toddses** [comment]
Since its not hurting anything at the moment, i'll sneak the correction into a future release.

**Pull request merged**